src/rosdiscover/recover/symbolic.py

```diff
@@ -28,6 +28,8 @@
 import typing as t
 
 from loguru import logger
+from ast import operator
```

Comment:
is this actually used? or has this accidentally been imported by your IDE?

Reply:
yep. removed